server/src/connectionHandlers/onHover.ts

```diff
@@ -47,6 +47,13 @@ export async function onHoverHandler (params: HoverParams): Promise<Hover | null
 
       hoverValue = `**${found.name}**\n___\n${found.definition}`
     }
+
+    // Find the exact variable with the same name and overrides
```

Comment:
I don't think you should sort with overrides. In the end, there is only one variable identified by the variable name. Overrides are just operators on it.

Reply:
In the context of the recipe, bitbake will only use the final value shown without override. These overrides can be useful for bitbake -e users who want to see the different overrides available but I don't think we need that complexity as long as we display all operations in "go to definitions", which is the case..

Reply:
For this specific case, there is no `BASE_LIB` without override from the output of `bitbake -e busybox`. Recipe is `busybox_1.36.1.bb`. Only found:
![image](https://github.com/yoctoproject/vscode-bitbake/assets/47988425/9569efa6-7a9f-4f23-99da-b5b5e0bd5f61)

Don't think the `BASE_LIB` and `BASELIB` are the same right?

Reply:
thye are different variables